routes/transaction.go

```diff
@@ -3580,6 +3617,26 @@ func (fes *APIServer) TransactionSpendingLimitFromResponse(
 		}
 	}
 
+	if len(transactionSpendingLimitResponse.AssociationLimitMap) > 0 {
+		transactionSpendingLimit.AssociationLimitMap = make(map[lib.AssociationLimitKey]uint64)
+		for _, associationLimitMapItem := range transactionSpendingLimitResponse.AssociationLimitMap {
+			appPKID := &lib.ZeroPKID
```

Comment:
Here, you default to the ZeroPKID. We should probably do the same above and default to the ZeroPublicKey.

Reply:
we need the PKID for the TransactionSpendingLimit Struct. TransactionSpendingLimitResponse gives us a public key and we need to convert it to a PKID for TransactionSpendingLimit. When we're going from TransactionSpendingLimit to TransactionSpendingLImitResponse, we convert the PKID to a public key